conceptnet5/vectors/evaluation/wordsim.py

```diff
@@ -3,9 +3,10 @@
 from conceptnet5.vectors.query import VectorSpaceWrapper
 import numpy as np
 import pandas as pd
-from scipy.stats import spearmanr
+from scipy.stats import spearmanr, tmean
 
-EVALS = ['men3000', 'rw', 'mturk', 'ws353', 'ws353-es', 'ws353-ro', 'gur350-de', 'zg222-de']
+EVALS = ['men3000', 'rw', 'mturk', 'ws353', 'ws353-es', 'ws353-ro', 'gur350-de', 'zg222-de', 'semeval2017-monolingual',
```

Comment:
Stylistic nitpick: This line goes on too long. I prefer wrapping to 80 columns when that's reasonable (it is here) or 100 when the extra space is necessary.